src/libraries/Directory.Build.targets

```diff
@@ -102,6 +102,12 @@
       <RuntimePath>$(NETCoreAppRuntimePackNativePath)</RuntimePath>
       <ItemName>NativeBinPlaceItem</ItemName>
     </BinPlaceTargetFrameworks>
+    <!-- Corresponding libMono and system.native libraries are added to testhost shared framework -->
```

Comment:
@mdh1418 @steveisok I guess this is a workaround until we get the self contained test runs story for Xamarin tests, right?

Reply:
Yes.  It will go away

Reply:
Awesome.